server/src/main/java/io/crate/execution/engine/collect/DocValuesAggregates.java

```diff
@@ -187,7 +197,8 @@
             }
             DocValueAggregator<?> docValueAggregator = ((AggregationFunction<?, ?>) func).getDocValueAggregator(
                 Symbols.typeView(aggregationReferences),
-                fieldTypes
+                fieldTypes,
+                literals.toArray(Literal[]::new)
```

Comment:
Could probably pass in the list directly without conversion. This is probably the only production call-site, so we don't need varargs call-convenience and using the list probably avoids some unchecked warnings